Sources/SwifterSwift/CoreLocation/CLVisitExtensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CLVisitExtensions.swift
+//  SwifterSwift
+//
+//  Created by Trevor Phillips on 09/01/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(CoreLocation) && (targetEnvironment(macCatalyst) || os(iOS))
+import CoreLocation
+
+// MARK: - Properties
+@available(iOS 8.0, *)
```

Comment:
I'm wondering if we need this check since the deployment target for the project is iOS8+

Reply:
+1
Btw, should we update that? I mean there are still projects with iOS 8-9 support? Maybe we can updato to 9/10 + :))

Reply:
It depends on how many times do we have `@available(iOS 9.0, *)` used in the library

Reply:
I can check that @omaralbeik , and if you give the thumbs up we can open an issue to move the target at to iOS 9 :)